### ISSUE TITLE
Make Hugo footnotes more accessible

### ISSUE THREAD

**Issue by baltpeter** [datenanfragen/website#239]
See: https://danielpost.com/articles/making-hugos-footnotes-accessible/

**Comment by baltpeter** [comment]
I would also really like to be able to see the footnote text without having to scroll to the bottom.

**Comment by baltpeter** [comment]
Maybe we can use [this](https://news.ycombinator.com/item?id=24747376) as a basis for footnote popups.

**Comment by baltpeter** [comment]
[This](https://github.com/goblindegook/littlefoot) might be another option, doesn't seem too huge.

**Comment by Cool-Runningz** [comment]
Hello! Would you be able to provide some more information as to where to find the current footnotes and/or a screenshot of what they currently look like?

I have some experience with accessibility and am interested in helping out with this if it is still available. Thanks!

**Comment by baltpeter** [comment]
@Cool-Runningz That would be great, thank you! I have assigned you to the issue.

One example of a page with footnotes is this one: https://www.datarequests.org/blog/honey-data-collection/ (source: https://github.com/datenanfragen/website/blob/master/content/en/blog/honey-data-collection/index.md?plain=1#L170)

![image](https://user-images.githubusercontent.com/4048809/135669180-a4a1cd59-71bb-435c-ba3a-43b0d940979c.png)

**Comment by Cool-Runningz** [comment]
@baltpeter Thank you for providing that information. If I may, I have a clarifying question:

- As far as implementation direction goes, would you prefer to go the route provided in the [article](https://danielpost.com/articles/making-hugos-footnotes-accessible/) you linked in your first comment? Or would you prefer to use the `littlefoot.js` library that was suggested?

**Comment by baltpeter** [comment]
@baltpeter After thinking about this some more, I think I actually prefer the approach from the [Hacker News comment](https://news.ycombinator.com/item?id=24747376). This is very lightweight, provides a good UX in my opinion and seems very semantically correct, which should be good for accessibility.

As we unfortunately can't influence the way Hugo renders the footnotes, you'd have to implement that in a similar way as described in the article using JS, though.

**Comment by zner0L** [comment]
And if you are unsure of your solution you can just open up a Draft PR and we can look at it.
